app/src/main/java/org/simple/clinic/home/overdue/OverdueScreenEvents.kt

```diff
@@ -22,4 +22,6 @@ data class AgreedToVisitClicked(val appointmentUUID: UUID) : UiEvent {
   override val analyticsName = "Overdue Screen:Appointment marked as patient-agreed-to-visit"
 }
 
-object RemoveFromListClicked : UiEvent
+data class RemoveFromListClicked(val appointmentUUID: UUID) : UiEvent {
+  override val analyticsName = "Overdue Screen:Appointment removed with reason"
```

Comment:
This name looks wrong. The appointment hasn't been removed yet.

Reply:
The action itself is called 'Remove from overdue list...' . Its named as the action + clicked.

Reply:
I think it could be named better. Printing `removed` to Heap would give incorrect results because the appointment hasn't been removed yet.

Reply:
Whoops, you're right. Fixed.

Reply:
Sorry, I was referring to the analytics name and not the class name :)